inspirehep/records/api/base.py

```diff
@@ -1098,3 +1102,18 @@ def get_earliest_date(self):
             date = earliest_date(dates, full_date=True)
             return date
         return None
+
+    def get_citations_count(self):
+        if self.pid_type in ["lit", "dat"]:
+            return len(self.model.citations)
+        raise CitationsNotSupportedForThisPidType
+
+    @property
+    def citation_count(self):
+        return self.get_citations_count()
+
+    def get_record_from_es(self):
```

Comment:
docstring?

Reply:
we have already such functionality on ES Literature factory

Reply:
It's a little different, I moved it to InspireSearch class.